tests/functional/spec/features/validation/date_validation/date-combined-mm-yyyy.spec.js

```diff
@@ -9,7 +9,54 @@ describe('Feature: Combined question level and single validation for MM-YYYY dat
   });
 
   describe('Period Validation', function () {
-    describe('Given I enter dates', function() {
+    describe('@watch Given I enter dates', function() {
+
+      it('When I enter a month but no year, Then I should see only a single invalid date error', function() {
+        return browser
+         .setValue(DateRangePage.dateRangeFromyear(), 2017)
+
+         .selectByValue(DateRangePage.dateRangeTomonth(), 6)
+         .setValue(DateRangePage.dateRangeToyear(), 2017)
+         .click(DateRangePage.submit())
+         .getText(DateRangePage.errorNumber(1)).should.eventually.contain('Enter a valid date')
+         .getText(DateRangePage.errorNumber(2)).should.eventually.not.contain("invalid literal for int() with base 10:");
```

Comment:
We should probably put a valid date in for `Period to` and check that there is only a single error. In other words there is no `DateRangePage.errorNumber(2)` element `isExisting(DateRangePage.errorNumber(2)).should.eventually.be.false`.
This should also be done for the other tests (lines 34 and 58).

Reply:
The To date is valid isn't it?  
To your other thing, yea, that's actually a better test.  I was thinking about it earlier that it might be a bit weak and what you've said confirmed it.